src/lib/constants.ts

```diff
@@ -19,3 +19,9 @@ export const API_ERROR = {
     code: 400
   }
 }
+
+export const SUPPORTED_NETWORKS: SupportedNetworks = {
+  bitcoin: networks.bitcoin,
+  testnet: networks.testnet,
+  regtest: Object.assign({}, networks.testnet, { bech32: 'bcrt' }) //Introduced in in newer versions of bitcoinjs-lib - https://github.com/bitcoinjs/bitcoinjs-lib/blob/489e96ca917bfff11357800344f1510b9ca68ce8/src/networks.js#L16
```

Comment:
to może trzeba uaktualnić wersję bitcoinjs?

Reply:
Tego nawet jeszcze nie ma w npm'ie. Miesiąc temu to dodali dopiero.